wflambda.go

```diff
@@ -0,0 +1,283 @@
+package wflambda
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"github.com/aws/aws-lambda-go/lambdacontext"
+	"github.com/rcrowley/go-metrics"
+	"github.com/wavefronthq/go-metrics-wavefront"
+	"log"
+	"os"
+	"reflect"
+	"strings"
+	"time"
+)
+
+type lambdaHandlerFunction func(context.Context, json.RawMessage) (interface{}, error)
+
+var (
+	server                  string
+	authToken               string
+	isReportStandardMetrics bool
```

Comment:
Drop the `is` prefix. It is idiomatic in go to just name `reportStandardMetrics`, `coldStart` etc. You can refer to https://golang.org/doc/effective_go.html#names for other naming conventions.